sandy/core/cov.py

```diff
@@ -812,6 +810,76 @@ def gls_update(self, S, Vy_extra, threshold=None):
         Vx_post = Vx_prior - A.dot(Vx_prior)
         return self.__class__(pd.DataFrame(Vx_post, index=index, columns=columns))
 
+    def _constrained_ls_sensitivity(self, S, threshold=None):

```

Comment:
can we call it `_constrained_gls_sensitivity` for consistency?

also, add a note and say that this method is equivalent to `_gls_sensitivity` but for a constrained system.